modules/accumulo/src/main/java/org/apache/fluo/accumulo/util/ColumnType.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.accumulo.util;
+
+import org.apache.accumulo.core.data.Key;
+
+public enum ColumnType {
+
+  TX_DONE, WRITE, DEL_LOCK, RLOCK, LOCK, ACK, DATA;
+
+  private long prefix;
+
+  /**
+   * @return The first possible timestamp
+   */
+  public long first() {
+    return prefix | ColumnConstants.TIMESTAMP_MASK;
+  }
+
+  public long prefix(long timestamp) {
```

Comment:
Could use `encode` for method name

Reply:
I made that change

Reply:
@keith-turner I just noticed you changed it to `enode` rather than `encode`.

Reply:
nice catch, @cjmctague !

Reply:
I opened #1061